spanner/spanner_snippets/snippet.go

```diff
@@ -688,6 +689,23 @@ func pgWriteWithTransactionUsingDML(ctx context.Context, w io.Writer, client *sp
 	return err
 }
 
+func pgAddStoringIndex(ctx context.Context, w io.Writer, adminClient *database.DatabaseAdminClient, database string) error {
```

Comment:
Is there a reason that the snippet needs to be added to this file as well?

Reply:
this is because of we need to support one liners in our getting-started documentation more information here https://github.com/GoogleCloudPlatform/golang-samples/pull/2493#issuecomment-1085659463

Reply:
Gotcha. Seems like a less-than-ideal pattern but that can be addressed some other time, not necessary for this PR.